app/models/group.rb

```diff
@@ -604,4 +604,12 @@ def limit_inheritance
   def self.with_one_coordinator
     published.select{ |g| g.admins.count == 1 }
   end
+
+  def default_cover_photo
+    if self.default_group_cover
+      /^.*(?=\?)/.match(self.default_group_cover.cover_photo.url).to_s
```

Comment:
Hi, this regex is confusing. Here's why it's there:

- By default, the paperclip url has an integer hash at the end of it, like `aws.com/test.jpg?10483202` (I assume this is a versioning / caching mechanism)
- When this gets transferred over to this url, it includes this hash, which then gets url encoded: `aws.com/test.jpg%3F10483202`
- This breaks everything

Loading the image seems to work okay without the version, and I'm honestly not that worried about whether we get the latest and greatest version of a default group logo; the point is that it's there and it's fine whatever it is.

Oh right, so this regex basically strips off everything after the first `?` in the url, leaving us with just `aws.com/test.jpg`, which loads up nicely.